ts/src/bitfinex2.ts

```diff
@@ -605,7 +605,7 @@ export default class bitfinex2 extends Exchange {
                         'max': this.parseNumber (maxOrderSizeString),
                     },
                     'price': {
-                        'min': this.parseNumber ('1e-8'),
+                        'min': undefined,
```

Comment:
Could you explain the reason of this change?

Reply:
Min price was hardcoded to '1e-8', however, sometime the exchange precision is inferior to 8 decimals. This was called by the trade controller to estimate min amounts, 1e-8 was rounded to 0 and was preventing the user to trade because of an error:
```
  // converts a price amount to the correct precision (round up)
  private priceToPrecision(cex: ccxt.Exchange, symbol: string, price: number): string {
    const market = cex.market(symbol);
    const result = cex.decimalToPrecision(
      price,
      ROUND_UP,
      market["precision"]["price"],
      cex.precisionMode,
      cex.paddingMode,
    );
    if (result === "0") {
      throw new Error(
        cex.id +
          " price of " +
          market["symbol"] +
          " must be greater than minimum price precision of " +
          String(market["precision"]["price"]),
      );
    }
    return result;
  }
```